csharp/ql/src/Bad Practices/Control-Flow/ConstantCondition.ql

```diff
@@ -72,6 +73,30 @@ class ConstantIfCondition extends ConstantBooleanCondition {
   }
 }
 
+/** A constant return value from a function with constant input expression. */
+class ConstantReturnValueCondition extends ConstantCondition {
```

Comment:
I think it would better to add this to the `isBooleanConstant` predicate inside `Completion.qll`; that way all control/data flow logic will benefit from this knowledge.